pkg/api/types.go

```diff
@@ -1380,3 +1381,15 @@ const (
 
 	PortHeader = "port"
 )
+
+// ResourceStats represents compute resource usage metrics associated with a pod or a container.
+type ResourceStats struct {
+	// Detailed instantaneous metrics.
+	Detailed *cadvisor_api.ContainerInfo `json:"detailed,omitempty"`
```

Comment:
Should we decouple what we expose in the master API from the cadvisor API? The downside is you would end up with two data structures and a bunch of stupid copying between them, but the advantage is that you could evolve the cadvisor API without impacting the master API.